hack/update-codegen.sh

```diff
@@ -38,6 +39,23 @@ ${CODEGEN_PKG}/generate-groups.sh "client,informer,lister" \
   "core:v1alpha1 upload:v1alpha1" \
   --go-header-file ${SCRIPT_ROOT}/hack/custom-boilerplate.go.txt
 
+
+# Workaround due to missing groupName in doc.go for CSI external snapshotter
+# See https://github.com/kubernetes-csi/external-snapshotter/issues/112
+set +e
+doc_go="${SCRIPT_ROOT}/vendor/github.com/kubernetes-csi/external-snapshotter/pkg/apis/volumesnapshot/v1alpha1/doc.go"
+status=`grep groupName $doc_go`
+
+if [[ -z "$status" ]]; then
+  sed -i '/^package/i \/\/ +groupName=snapshot.storage.k8s.io' $doc_go
```

Comment:
I think it would be best to check in the modified file otherwise running `make generate` will leave give changes in working directory.  Alternatively, the github issue above has been closed.  So maybe can vendor in a more recent version?

Reply:
The fix is currently in master only, not on a released version.
If we check in the modified file, after "make deps-update" it will be overridden, and eventually checked in back without the fix.
Possible solutions:
1 - undo the change after generating in the bash script
2 - use master version of the external-snapshot
3 - keep current solution until we get an updated version of the CSI library.

What do you think?

Reply:
I think I am okay with 1) or 2).  If you go with 2, pin to a specific commit.  What do you think @awels ?

Reply:
I am fine with 2 or 3, we just somehow need to make sure we update it to a released version as soon as it is released.  I think I prefer 2 with a pinned commit, to avoid leaving stuff around in certain cases.

Reply:
OK, pinned to commit and removed ugly hack.